libs/common/ui/src/components/Card/CardThemed.tsx

```diff
@@ -18,5 +18,7 @@ export const CardThemed = styled(Card, {
       ? theme.palette.contentLight.main
```

Comment:
could you make this a separate function with if else/switch case and return statements? nested  nested nested ternary is impossible to read